src/pages/Filing/FilingApp/FileSubmission.tsx

```diff
@@ -79,10 +79,15 @@ export function FileSubmission(): JSX.Element {
 
   // TODO compare lei and filing period to getlastsubmission before updating object
   useEffect(() => {
-    if (actualDataGetSubmissionLatest) {
+    if (!isFetchingGetSubmissionLatest && !errorGetSubmissionLatest) {
+      setInitialGetSubmissionLatestFetched(true);
       setDataGetSubmissionLatest(actualDataGetSubmissionLatest);
     }
-  }, [actualDataGetSubmissionLatest]);
+  }, [
+    actualDataGetSubmissionLatest,
+    isFetchingGetSubmissionLatest,
+    errorGetSubmissionLatest,
+  ]);
```

Comment:
@sigmund.tang 

It seems that with the check `if (actualDataGetSubmissionLatest)` you are trying to ensure that the call to `/submission/latest` is complete and successful, then setting some flags in the local state to remove the `Loading` indicator? 
 
However, for new Filers who have not uploaded any submissions yet, `actualDataGetSubmissionLatest` remains empty/falsy, so the `initialGetSubmissionLatestFetched` flag stays `false` and the `Loading` indicator remains displayed.